### ISSUE TITLE
UI improvements

### ISSUE THREAD

**Issue by rossrowe** [saucelabs/bamboo_sauce#3]
From a user's support request:

The frame window is quite small, I would prefer if the links just opened a new browser window or new tab. Also it would be nice if the sauce lab results appeared as a link on the build results "Tests" screen alongside the failed tests that are being reported.

**Comment by foal** [comment]
Yes!  I also prefer just a link. There is impossible to see screenshot (no scroll bar). I think the reason is design update on sauce lab side.
